canonical-web-and-design/jaas-dashboard
# Refactor model details

## Done

- Move `pages > Models > Details` to `pages > ModelDetails`
- Move utility functions out of `generators.js` and into `utils.js`

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check nothing is on fire
